usr/local/share/bastille/create.sh

```diff
@@ -261,6 +263,13 @@ create_jail() {
             for _link in ${LINK_LIST}; do
                 ln -sf /.bastille/${_link} ${_link}
             done
+            # Properly link shared ports on thin jails in read-write.
+            if [ -d "${bastille_releasesdir}/${RELEASE}/usr/ports" ]; then
+                if [ ! -d "${bastille_jail_path}/usr/ports" ]; then
+                    mkdir ${bastille_jail_path}/usr/ports
+                fi
+                echo -e "${bastille_releasesdir}/${RELEASE}/usr/ports ${bastille_jail_path}/usr/ports nullfs rw 0 0" >> "${bastille_jail_fstab}"
```

Comment:
don't know if we need to make this a default. is easily templated however.

Reply:
Hi cedwards, this is just the default and how it works on Thickjail, if the user select ports initially when `bootstrapped` they should be presented in the jail seamlessly, however since the Thinjail nature and they have a shared base across the jails, this a basic requirement to map the wanted ports by the user on the jails by default, also this is only created if the `release/usr/ports` exist, just the same way the shared base is applied but with `read-write` permissions as expected.

IMO, this should be an one time handled `bootstrap` jail-built procedure and should be handled internally rather tan tempting basic requirements, between always up for suggestions.

Regards